--- a/contrib/60-lirc.rules	2016-10-27 07:11:14.000000000 -0500
+++ b/contrib/60-lirc.rules	2023-01-19 16:07:55.910941592 -0600
@@ -1,9 +1,9 @@
 # Make the /dev/lirc* devices accessible for users in the group "lirc"
 # using regular group permissions.
 
-KERNEL=="lirc[0-9]*", SUBSYSTEM=="lirc", GROUP="lirc", MODE="0660"
+KERNEL=="lirc[0-9]*", SUBSYSTEM=="lirc", GROUP="_lirc", MODE="0660"
 
 # Grant rw permission to the lirc group for USB devices using acl(5).
 
 ACTION=="add", SUBSYSTEM=="usb", ENV{DEVNAME}=="?*", \
-    RUN+="/usr/bin/sh -c '/usr/bin/setfacl -m g:lirc:rw %E{DEVNAME} 2>/dev/null || :'"
+    RUN+="/usr/bin/sh -c '/usr/bin/setfacl -m g:_lirc:rw %E{DEVNAME} 2>/dev/null || :'"
